central/graphql/resolvers/clusters_postgres_test.go

```diff
@@ -0,0 +1,161 @@
+//go:build sql_integration
+
+package resolvers
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stackrox/rox/central/cluster/datastore"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	"github.com/stackrox/rox/pkg/fixtures"
+	"github.com/stackrox/rox/pkg/postgres/pgtest"
+	"github.com/stackrox/rox/pkg/protoassert"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestGraphQLClustersPostgres(t *testing.T) {
+	suite.Run(t, new(graphQLClusterTestSuite))
+}
+
+type graphQLClusterTestSuite struct {
+	suite.Suite
+
+	testDB   *pgtest.TestPostgres
+	resolver *Resolver
+
+	clusters []*storage.Cluster
+
+	scopeObjects []*v1.ScopeObject
+}
+
+func (s *graphQLClusterTestSuite) SetupSuite() {
+	s.testDB = pgtest.ForT(s.T())
+
+	clusterDataStore, err := datastore.GetTestPostgresDataStore(s.T(), s.testDB)
+	s.Require().NoError(err)
+
+	s.resolver = &Resolver{ClusterDataStore: clusterDataStore}
+
+	ctx := sac.WithAllAccess(context.Background())
+	for i := 0; i < 6; i++ {
+		cluster := fixtures.GetCluster(fmt.Sprintf("Test cluster %d", i+1))
+		id, addErr := clusterDataStore.AddCluster(ctx, cluster)
+		s.Require().NoError(addErr)
+		cluster.Id = id
+		s.clusters = append(s.clusters, cluster)
+		s.addClusterScopeObject(cluster)
+	}
+}
+
+func (s *graphQLClusterTestSuite) addClusterScopeObject(cluster *storage.Cluster) {
+	scopeObject := &v1.ScopeObject{
+		Id:   cluster.Id,
+		Name: cluster.Name,
+	}
+	s.scopeObjects = append(s.scopeObjects, scopeObject)
+}
+
+func (s *graphQLClusterTestSuite) TearDownSuite() {
+	s.testDB.Teardown(s.T())
+}
+
+func (s *graphQLClusterTestSuite) TestClustersForPermission() {
```

Comment:
I'm wondering if we should test calls on `ComplianceClusters` - in that case, we are covering almost the full flow for this query. And also test cases with no `resources.Compliance` access would make more sense.